README.md

```diff
@@ -79,7 +90,17 @@ var express = require("express");
 var app = express();
 
 // Your code starts here...
+app.get("/something/:name", function(req, res){
+  console.log(req.params.name);
+
+app.post('something', function(req, res))
 
+app.use(bodyParser.json());
+
+app.listen(3001, function(){
+  console.log("We are in business")
+})
+})
```

Comment:
hmm, what is the app.use and app.listen for here? It looks like everything is contained within your app.get which won't work.  We are looking for app.get, app.post, app.put, and app.delete